### PR TITLE
Add Kubernetes Legacy Package Repositories Will Be Frozen On September 13, 2023 blog post

### PR DESCRIPTION
**Pull request by xmudrii**
This PR adds a new blog post "Kubernetes Legacy Package Repositories Will Be Frozen On September 13, 2023" to describe the change that we announced today: https://groups.google.com/g/kubernetes-sig-release/c/akDpTHZsv_U/m/X2EPRFuVFQAJ

The blog post release date is set to tomorrow, September 1st, 2023. This is a tentative date, we are pending the confirmation, but given the short notice, it would be preferred to publish this post as soon as possible.

Huge thanks to everyone involved in writing this blog post, especially to @sftim for pushing everything forward on such a short notice.

/assign @sftim @jeremyrickard 
cc @kubernetes/release-engineering

### CODE REVIEW

File: content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+layout: blog
+title: "Kubernetes Legacy Package Repositories Will Be Frozen On September 13, 2023"
+date: 2023-09-01
```
Comment (sftim) [inline content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md:4]:
```suggestion
2023-08-31T16:30:00-07:00
```

**Comment by xmudrii** [comment]
cc-ing folks that are mentioned as authors and that helped with writing this blog post: @mrbobbytables @chris-short (@sftim and @jeremyrickard are already pinged above)

File: content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+slug: legacy-package-repository-deprecation
+---
+
+**Authors**: Bob Killen (Google), Chris Short (AWS), Jeremy Rickard (Microsoft), Marko Mudrinić (Kubermatic), Tim Bannister (The Scale Factory)
+
+On August 15, 2023, the Kubernetes project announced the general availability of the community-owned package repositories for Debian and RPM packages available at `pkgs.k8s.io`. The new package repositories are replacement for the legacy Google-hosted package repositories: `apt.kubernetes.io` and `yum.kubernetes.io`. The [announcement blog post for `pkgs.k8s.io](/blog/2023/08/15/pkgs-k8s-io-introduction/) highlighted that we will stop publishing packages to the legacy repositories in the future.
```
Comment (sftim) [inline content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md:10]:
```suggestion
On August 15, 2023, the Kubernetes project announced the general availability of the community-owned package repositories for Debian and RPM packages available at `pkgs.k8s.io`. The new package repositories are replacement for the legacy Google-hosted package repositories: `apt.kubernetes.io` and `yum.kubernetes.io`. The [announcement blog post for `pkgs.k8s.io`](/blog/2023/08/15/pkgs-k8s-io-introduction/) highlighted that we will stop publishing packages to the legacy repositories in the future.
```

File: content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md (hunk 1 above)
Comment (xmudrii) [reply content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md]: Fixed 👍

**Review by sftim — COMMENTED** [review]
/lgtm
/approve
/hold

One nit

File: content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md (hunk 1 above)
Comment (xmudrii) [reply content/en/blog/_posts/2023-09-01-legacy-package-repository-deprecation.md]: Also fixed the file name

**Comment by sftim** [comment]
Ship it!

/lgtm
/hold cancel

**Review by Verolop — COMMENTED** [review]
/lgtm

**Pull request merged**